people.md

```diff
@@ -1,93 +1,84 @@
-# Steering Council and Institutional Partners
-
-Names are followed by GitHub usernames.
-
-## Benevolent Dictator for Life
-
-Fernando Perez (@fperez) is the Benevolent Dictator for Life (BDFL).
-
-## Steering Council
-
-Alphabetical by first name:
-
-- Afshin Darian, @afshin (2018)
-- Ana Ruvalcaba, @Ruv7 (2018)
-- Brian Granger, @ellisonbg (Founding member, 2014)
-- Carol Willing, @willingc (2017)
-- Damian Avila, @damianavila (Founding member, 2014)
-- Fernando Perez, @fperez (Founding member, 2014)
-- Jason Grout, @jasongrout (2015)
-- Jessica Hamrick, @jhamrick (2015)
-- Kyle Kelley, @rgbkrk (Founding member, 2014)
-- M Pacer, @mpacer (2018)
-- Matthias Bussonnier, @carreau (Founding member, 2014)
-- Min Ragan-Kelley, @minrk (Founding member, 2014)
-- Paul Ivanov, @ivanov (2016)
-- Peter Parente, @parente (2016)
-- Steven Silvester, @blink1073 (2016)
-- Sylvain Corlay, @sylvaincorlay (2016)
-- Thomas Kluyver, @takluyver (Founding member, 2014)
-
-### Retired steering council members
-
-- Jonathan Frederic, @jdfreder (2016-2018)
-
-### NumFOCUS Subcommittee
-
-- Ana Ruvalcaba, @Ruv7
-- Fernando Perez, @fperez
-- Matthias Bussonnier, @carreau
-- Stefan van der Walt (non-council member), @stefanv
-- Thomas Kluyver, @takluyver
-
+# Leadership Directory
+
+This is a directory of the present and past leadership of Project Jupyter.
+## [Executive Council](executive_council.md)
+
+Alphabetical by first name, names are followed by GitHub usernames.
+
+- Afshin Darian, [@afshin](https://github.com/afshin)
+- Brian Granger, [@ellisonbg](https://github.com/ellisonbg)
+- Fernando Perez, [@fperez](https://github.com/fperez)
+-
+-
+-
+
+## [Software Steering Council](software_steering_council.md)
+
+| Subproject | Representative | GitHub username |
+| ---------- | -------------- | --------------- |
+| Jupyter Accessibility | Isabela Presedo-Floyed | [@isabela-pf](https://github.com/isabela-pf) |
+| Jupyter Foundations | | |
```

Comment:
We just finished the Foundations SSC election, and Paul was chosen as the representative.

```suggestion
| Jupyter Foundations | Paul Ivanov | [@ivanov](https://github.com/ivanov) |
```